main.py

```
@@ -450,6 +487,34 @@ async def cli_loop(sk, pk, chain, mempool, network):
+            if len(parts) < 2:
+                print("  Usage: import_key <hex_private_key>")
+                continue
+            
+            try:
+                new_sk = SigningKey(bytes.fromhex(parts[1]))
+                new_pk = new_sk.verify_key.encode(encoder=HexEncoder).decode()
+                
+                path = datadir or "."
+                keystore_path = os.path.join(path, "keystore.json")
+                os.makedirs(path, exist_ok=True)
+                with open(keystore_path, "w") as f:
```

Ruff (0.15.20)

[warning] Async functions should not open files with blocking methods like `open`

(ASYNC230)